scripts/rv_expression_association/README.md

```diff
@@ -0,0 +1,15 @@
+This script counts how many rare variants (freq<5%) in the TOB-WGS dataset there are.
+The filters applied are:
+* QC
+* biallelic SNVs
+* not HomRef
+* freq<5%
+
+To run:
+```
+analysis-runner --dataset "tob-wgs" \
+    --description "count TOB rare variants" \
+    --access-level "standard" \
+    --output-dir "v0" \
```

Comment:
It doesn't matter much in this particular case, but please use something more descriptive than that for the top-level directory.

Reply:
just in the command or the README more generally?